### PR TITLE
Fixed the issue with formatting issues not being reported when verification fails

### PR DESCRIPTION
**Pull request by wasylkowski**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by eparis** [comment]
This does not appear to be a problem on my system. What problem is this supposed to fix?
```
$ go version
go version go1.8.1 linux/amd64

$ hack/verify-gofmt.sh 
diff ./cmd/genman/gen_kube_man.go gofmt/./cmd/genman/gen_kube_man.go
--- /tmp/gofmt337461115	2017-06-12 09:13:10.817350914 -0400
+++ /tmp/gofmt682044062	2017-06-12 09:13:10.817350914 -0400
@@ -38,7 +38,7 @@
 
 func main() {
 	// use os.Args instead of "flags" because "flags" will mess up the man pages!
-	  path := "docs/man/man1"
+	path := "docs/man/man1"
 	module := ""
 	if len(os.Args) == 3 {
 		path = os.Args[1]

$ echo $?
1

```

**Comment by wasylkowski** [comment]
This is indeed very interesting, because after I fixed formatting issues I have had, and added a formatting issue given by you, I get the same (working) behavior without the fix. Let me look more deeply into why this is so

**Comment by wasylkowski** [comment]
OK, I have found it. If the file happens not to parse correctly, verify_gofmt spits out a a not very useful message:
`
$ hack/verify-gofmt.sh
!!! Error in hack/verify-gofmt.sh:52
  Error in hack/verify-gofmt.sh:52. 'diff=$(find_files | xargs ${gofmt} -d -s 2>&1)' exited with status 123
Call stack:
  1: hack/verify-gofmt.sh:52 main(...)
Exiting with status 1
`

After my fix you get the following:
`
$ hack/verify-gofmt.sh
./test/e2e/stackdriver_monitoring.go:95:100: missing ',' in argument list
`

I believe the fix makes verify_gofmt more useful, but it is up to you to agree/disagree with me. Let me know what you think.

**Comment by wasylkowski** [comment]
Of course I will change the comment in the file.

**Comment by wasylkowski** [comment]
Comment updated.

File: hack/verify-gofmt.sh [hunk 1]
```diff
@@ -46,9 +46,12 @@ find_files() {
         -o -wholename '*/bindata.go' \
       \) -prune \
     \) -name '*.go'
-}
```
Comment (eparis) [inline hack/verify-gofmt.sh]: Where did this go?

**Comment by cblecker** [comment]
With these changes, if the script is unable to parse the file and this new error is displayed, does the whole script itself still exit non-zero?

**Comment by eparis** [comment]
It does. I double checked.

File: hack/verify-gofmt.sh (hunk 1 above)
Comment (wasylkowski) [inline hack/verify-gofmt.sh]: Oops. Sorry about that. Brought it back.

**Comment by cblecker** [comment]
@k8s-bot ok to test
/release-note-none

**Comment by eparis** [comment]
/lgtm

although I think this is going to have to wait for the queue to reopen for 1.8.

**Comment by wasylkowski** [comment]
Thanks Eric!

**Comment by mwielgus** [comment]
@k8s-bot test this please

**Pull request merged**